config/danbooru_default_config.rb

```diff
@@ -649,14 +649,47 @@ def enable_signups?
+          {
+            name: 'dnp_artist',
+            reason: "The artist of is on the [[avoid_posting|avoid posting list]]",
+            text: "Certain artists have requested that their work is not to be published on this site, and were granted [[avoid_posting|Do Not Post]] status.\nSometimes, that status comes with conditions; see [[conditional_dnp]] for more information"
+          },
+          {
+            name: 'pay_content',
+            reason: "Paysite, commercial, or subscription content",
+            text: "We do not host paysite or commercial content of any kind. This includes Patreon leaks, reposts from piracy websites, and so on."
+          },
+          {
+            name: 'trace',
+            reason: "Trace of another artist's work",
+            text: "Images traced from other artists' artwork are not accepted on this site. Referencing from something is fine, but outright copying someone else's work is not.\nPlease, leave more information in the comments, or simply add the original artwork as the posts's parent if it's hosted on this site."
+          },
+          {
+            name: 'previously_deleted',
+            reason: "Previously deleted",
+            text: "Posts usually get removed for a good reason, and reuploading of deleted content is not acceptable.\nPlease, leave more information in the comments, or simply add the original post as this post's parent."
+          },
+          {
+            name: 'real_porn',
+            reason: "Real-life pornography",
+            text: "Posts featuring real-life pornography are not acceptable on this site. No exceptions.\nNote that images featuring non-erotic photographs are acceptable."
+          },
+          {
+            name: 'corrupt',
+            reason: "File is either corrupted, broken, or otherwise does not work",
+            text: "Something about this post does not work quite right. This may be a broken video, or a corrupted image.\nEither way, in order to avoid confusion, please explain the situation in the comments."
+          },
+          {
+            name: 'inferior',
+            reason: "Duplicate or inferior version of another post",
+            text: "A superior version of this post already exists on the site.\nThis may include images with better visual quality (larger, less compressed), but may also feature \"fixed\" versions, with visual mistakes accounted for by the artist.\nNote that edits and alternate versions do not fall under this category.",
+            parent: true
+          },
       ]
     end
+    def flag_reason_48hours
+      "If you are the artist, and want this image to be taken down [b]permanently[/b], file a \"takedown\":/static/takedowns instead.\nTo replace the image with a \"fixed\" version, upload that image first, and then use the \"Duplicate or inferior version\" reason above.\nFor accidentally released paysite or private content, use the \"Paysite, commercial, or private content\" reason above."
```

Comment:
```suggestion
      "If you are the artist, and want this image to be taken down [b]permanently[/b], file a \"takedown\":/static/takedown instead.\nTo replace the image with a \"fixed\" version, upload that image first, and then use the \"Duplicate or inferior version\" reason above.\nFor accidentally released paysite or private content, use the \"Paysite, commercial, or private content\" reason above."
```
The current link 404s, the same is true for the wiki page template. Once that's fixed this should be all good.

Reply:
Should be fixed in 0c99e1209507bf633783b3e2b8214ef0ec50b855.